west.yml

```diff
@@ -129,6 +129,13 @@ manifest:
       path: modules/alexa-embedded
       revision: face92d8c62184832793f518bb1f19379538c5c1
       remote: alexa
+    - name: sdk-homekit
+      path: homekit
+      revision: pull/17/head
+      groups:
+      - homekit
+
+  group-filter: [-homekit]
```

Comment:
Should we place this at the top, before `projects:`?

Reply:
In doc snippet it is under projects https://docs.zephyrproject.org/latest/guides/west/manifest.html?highlight=west%20manifest#project-groups-and-active-projects
But I can move it to the top

Reply:
Moved on top